src/niscope/examples/niscope_fetch_forever.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
+
+import argparse
+import datetime
+import niscope
+import numpy as np
+import pprint
+import sys
+
+
+pp = pprint.PrettyPrinter(indent=4, width=80)
+
+
+# This example is based on the C FetchForever example
+def example(resource_name, channels, options, total_acquisition_time_in_seconds, voltage, sample_rate_in_hz, samples_per_fetch):
+    total_samples = int(total_acquisition_time_in_seconds * sample_rate_in_hz)
+    channel_list = channels.split(',')  # We need channels as a list
```

Comment:
Can the user pass `channels` as empty string to signify "all"?
Can the user pass `channels` using ranges, such as "0-4"?
What if the user passes `channels` but adds whitespace such ass "0, 1, 2, 3"?

Seems like those will make this fail in unexpected ways. Why do the channels need to be passed in at all? Can we assume "all channels" and get those from the session instead?

Reply:
Other examples allow passing in channel list

Removed